docs-chef-io/content/supermarket/supermarket_upgrade.md

```diff
@@ -3,103 +3,181 @@ title = "Upgrade Supermarket"
 date = 2021-12-28T11:04:48-08:00
 draft = false
 gh_repo = "supermarket"
-publishDate = 2022-01-03
 
 [menu]
   [menu.supermarket]
     title = "Upgrades"
     identifier = "supermarket/server/upgrade.md"
     parent = "supermarket/server"
-    weight = 25
 +++
 
-
+<!-- markdownlint-disable MD033 -->
 ## Upgrade Matrix
 
-Running Version| Upgrade Version | Supported Version
----------|----------|---------
- A1 | B1 | C1
- A2 | B2 | C2
- A3 | B3 | C3
+  If running Supermarket 4.2, you can upgrade directly to the latest releases of Supermarket 4.3. If you are running a release with version less than 4.2 you must perform a stepped upgrade as outlined below.
 
-## Upgrade Supermarket
+Running Version | Upgrade Version | Supported Version
+----------------|-----------------|------------------
+4.2             | 4.3             | No
+4.1             | 4.2             | No
+4.0             | 4.2             | No
+< 4.0           | 4.2             | No
 
-### Database Preparation
+## Supported Release
 
-### Upgrade Steps
+Chef Supermarket uses the PostgreSQL database. [PostgreSQL 9.3 is EOL](https://endoflife.date/postgresql) and Private Supermarket users should upgrade to [Supermarket 4.3](https://www.chef.io/downloads/tools/supermarket) or above and migrate to [Postgres 13](https://www.postgresql.org/about/news/postgresql-13-released-2077/).
 
-## External PostgreSQL
+Chef Software supports Supermarket 4.3 release and later. Earlier releases are not supported. For more information about supported Chef Software see the [Supported Versions](https://docs.chef.io/versions/#supported-commercial-distributions) documentation.
 
-The External PostgreSQL upgrade steps are provided as a courtesy. It is the responsibility of the user to upgrade and maintain any External PostgreSQL configurations.
+## Upgrade a Private Supermarket
 
-Follow these steps to upgrade the PostgreSQL major version.
+Every Private Supermarket installation is unique. These are general steps for upgrading a Private Supermarket.
 
-## Upgrade Supermarket
+  1. Shut down the server running Private Supermarket.
```

Comment:
This makes it sound like we need to power off the system. We need to stop the services

Reply:
Corrected language.